monai/networks/nets/swin_unetr.py

```diff
@@ -1055,3 +1055,35 @@ def forward(self, x, normalize=True):
         x4 = self.layers4[0](x3.contiguous())
         x4_out = self.proj_out(x4, normalize)
         return [x0_out, x1_out, x2_out, x3_out, x4_out]
+
+
+def filter_swinunetr(key, value):
```

Comment:
We should have tests with this function. If possible a code example of using this with a default network would help as well since just mentioning `copy_model_state` doesn't really say enough about use case.

Reply:
Looks good, thanks!